src/pardibaal/DBM.cpp

```diff
@@ -78,25 +73,28 @@ namespace pardibaal {
 
     bool DBM::is_empty() const {
```

Comment:
You probably want to add a flag for "check only cached", or maybe a function.
I.e. some places you only want to do a cheap empty-check, as the empty-computation is as expensive as the normal operation (n^2).